server/src/main/kotlin/core/pokerhands/StraightFlush.kt

```diff
@@ -0,0 +1,28 @@
+package core.pokerhands
+
+import core.Card
+import core.CardRank
+
+class StraightFlush(cards: List<Card>) : PokerHand(HandRank.STRAIGHT_FLUSH, cards) {
```

Comment:
Maybe we should extend Straight class as it uses the same compareWithinRank function.

Reply:
Idk it souls be ugly

Reply:
Maybe we should consider implementing some sort of compareStraight function in utils to avoid code duplication?